### PR TITLE
MSS-1064: gallery lab fixes

### PR DESCRIPTION
**Pull request by frankie297**
1. Reinstated the ad slot, and fixed the margin
2. Made the related content container the same colour as the article body to make it more consistent
3. Add some color and underline to links (using this article as a guideline (App version not dotcom): https://www.theguardian.com/rdg-epic-rail-journeys/2019/aug/01/views-box-set-binges-and-reducing-co2-nine-reasons-to-love-train-travel) 

| Before | After |
| --- | --- |
|||

| Before | After |
| --- | --- |
|||

| Before | After |
| --- | --- |
|||

### CODE REVIEW

File: ArticleTemplates/assets/scss/modules/_related-content.scss [hunk 1]
```diff
@@ -9,4 +9,8 @@
         width: 1200px;
     }
  }
- 
\ No newline at end of file
+
+ .is_advertising.garnett--type-guardianlabs[data-content-type='gallery'] .related-content {
+    background: color(global-adv-bg);
```
Comment (webb04) [inline ArticleTemplates/assets/scss/modules/_related-content.scss]: I was thinking of making the "paid for by" section this colour and keeping related content how it is?

File: ArticleTemplates/assets/scss/type/_media-gallery.scss [hunk 2]
```diff
@@ -1,5 +1,9 @@
 .garnett--type-media[data-content-type='gallery'],
 .is_advertising.garnett--type-guardianlabs[data-content-type='gallery'] .article {
+    .advert-slot--mpu {
+        margin: 0;
```
Comment (webb04) [inline ArticleTemplates/assets/scss/type/_media-gallery.scss]: I think ads on other pages have a top/bottom margin

File: ArticleTemplates/assets/scss/type/_media-gallery.scss [hunk 3]
```diff
@@ -219,6 +223,15 @@
 }
 
 .is_advertising.garnett--type-guardianlabs[data-content-type='gallery'] .article {
+    p a, em a {
+        color: #5fa794;
```
Comment (webb04) [inline ArticleTemplates/assets/scss/type/_media-gallery.scss]: Do we have a SASS variable for this colour?

File: ArticleTemplates/assets/scss/type/_media-gallery.scss (hunk 2 above)
Comment (frankie297) [reply ArticleTemplates/assets/scss/type/_media-gallery.scss]: I added the same top and bottom margin as other adverts but i didn't know if it looked a bit strange with the gap at the bottom? 
Simulator Screen Shot - iPhone Xʀ - 2019-08-20 at 15 00 30

File: ArticleTemplates/assets/scss/type/_media-gallery.scss (hunk 3 above)
Comment (frankie297) [reply ArticleTemplates/assets/scss/type/_media-gallery.scss]: We did not have a SASS variable, i will add one 👍

File: ArticleTemplates/assets/scss/modules/_related-content.scss (hunk 1 above)
Comment (frankie297) [reply ArticleTemplates/assets/scss/modules/_related-content.scss]: oh sorry, I will change that, i must have misunderstood on the ticket

Comment (frankie297) [reply ArticleTemplates/assets/scss/modules/_related-content.scss]: - now looks like this

File: ArticleTemplates/assets/scss/style-async.scss [hunk 4]
```diff
@@ -15,6 +15,7 @@
 @import 'modules/inline-list';
 @import 'modules/interactive';
 @import 'modules/more';
+@import 'type/media-gallery';
```
Comment (webb04) [inline ArticleTemplates/assets/scss/style-async.scss]: I think this is already included in `style-sync.scss`?

File: ArticleTemplates/assets/scss/type/_media-gallery.scss (hunk 2 above)
Comment (webb04) [reply ArticleTemplates/assets/scss/type/_media-gallery.scss]: Yeah maybe 
```
margin: 0;
margin-bottom: 32px;
```

So we have the same margin above and below the ad?

**Comment by webb04** [comment]
Just spotted a couple of cases where we might need additional styles: 
- We might need left padding for iPads
- The ads on glab galleries aren't clearing to go underneath the images

**Pull request merged**